### ISSUE TITLE
[Test] `check_style` and `run_tests` do not work on Python 2.

### ISSUE THREAD

**Issue by tjprescott** [Azure/azure-cli#4840]
Related to https://github.com/Azure/azure-cli/issues/3883

When I run these scripts I get:
```
Traceback (most recent call last):
  File "C:\Users\trpresco\Documents\github\env2\Scripts\check_style-script.py", line 11, in <module>
    load_entry_point('azure-cli-dev-tools', 'console_scripts', 'check_style')()
  File "C:\Users\trpresco\Documents\github\env2\lib\site-packages\pkg_resources\__init__.py", line 561, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "C:\Users\trpresco\Documents\github\env2\lib\site-packages\pkg_resources\__init__.py", line 2626, in load_entry_point
    raise ImportError("Entry point %r not found" % ((group, name),))
ImportError: Entry point ('console_scripts', 'check_style') not found
```

```
Traceback (most recent call last):
  File "C:\Users\trpresco\Documents\github\env2\Scripts\run_tests-script.py", line 11, in <module>
    load_entry_point('azure-cli-dev-tools', 'console_scripts', 'run_tests')()
  File "C:\Users\trpresco\Documents\github\env2\lib\site-packages\pkg_resources\__init__.py", line 561, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "C:\Users\trpresco\Documents\github\env2\lib\site-packages\pkg_resources\__init__.py", line 2626, in load_entry_point
    raise ImportError("Entry point %r not found" % ((group, name),))
ImportError: Entry point ('console_scripts', 'run_tests') not found
```

---

### Environment summary

Install Method (e.g. pip, interactive script, apt-get, Docker, MSI, edge build) / CLI version (`az --version`) / OS version / Shell Type (e.g. bash, cmd.exe, Bash on Windows)  

dev_setup.py / azure-cli (2.0.21) / Win 10 / CMD.exe
```
```

**Comment by derekbekoe** [comment]
This actually looks related to https://github.com/Azure/azure-cli/commit/b5e53a8d44977337aeb85d52bbf0833f4756b3b9#diff-2071ffe9914306221414bd4300c8a8a3R52.

I'm surprised as it's fully documented at http://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point.

What's the full Python version & pip version you're using?

**Comment by tjprescott** [comment]
Python 2.7.11 pip 9.0.1

**Comment by tjprescott** [comment]
Verified that this is fixed as of 2.0.24 (and probably was earlier than that).
